app/assets/themes/fearlesscities/_body.scss

```
@@ -0,0 +1,40 @@
+@import "variables";
+
+body {
+  font-family: 'Lora', 'Poppins', sans-serif;
+  letter-spacing: 0.5px;
+}
+
+h1, .h1,
+h2, .h2,
+h3, .h3,
+h4, .h4,
+h5, .h5,
+h6, .h6 {
+  font-family: "Lora", sans-serif;
```

What would you think of using this other font for headings?

```suggestion
  font-family: "Poppins Bold", sans-serif;
```

---

Yes, we can change it!